conda/recipes/libraft/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+./build.sh libraft tests bench -v --allgpuarch --compile-dist --compile-nn --no-nvtx
```

Comment:
@Ethyling, I need to think about this one a little bit. One of the reasons we were running the compile 3 separate times was because a user should be able to install the headers conda package without the distances or nn packages. They should also be able to install the headers and then install either the distances or the nn packages at a later time and the relevant cmake files should allow for this. 

@robertmaynard do you see any reason compiling them all at once and installing the components separately would create a problem here?